mozita.py

```diff
@@ -29,7 +29,7 @@ def risposte(msg):
 
     start = InlineKeyboardMarkup(inline_keyboard=[
                     [InlineKeyboardButton(text='Mostrami che cosa posso fare', callback_data='/azioni')],
```

Comment:
modificare /azioni in /help (verificare TUTTO il codice. Dove c'è "/azioni" sostituire con "help"

Reply:
(modifico io)